bin/copyProjectDataToPrm.sh

```diff
@@ -488,6 +488,7 @@ else
 					log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "${project}/${run} calculateProjectMd5s not yet finished."
 				fi
 			done
+			rm -f "${PRM_ROOT_DIR}/Samplesheets/${project}.${SAMPLESHEET_EXT}"
```

Comment:
This will not always work as expected. You may have one or more runs that are not "ready" yet resulting in: 
```
else
    log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "${project}/${run} calculateProjectMd5s not yet finished."
```
and then you nevertheless rm the project samplesheet. That should only happen when everything for a project was done.

Reply:
well, it is archived.. 
If there will be a run02 in the future, there needs to be a samplesheet after all.. You have a copy in the archive and you can copy it manually from the archive to Samplesheets. run02 can't happen without copying the samplesheet to the samplesheets folder after all.
If you don't remove the samplesheet now, it will never happen (and i was kinda annoyed by the fact that there are dozens of samplesheets not being archived).